crates/node_binding/src/tokio_runtime.rs

```diff
@@ -0,0 +1,15 @@
+use once_cell::sync::Lazy;
+
+/// TODO(hyf0): we might need a way to cleanup this resource
```

Comment:
Nit: remove the comment if you are unsure about the problem, it's not usable for readers.